https://github.com/libxls/libxls/pull/118

From f46aea10c50fb08657c15f8f3e813d5434b8b4d5 Mon Sep 17 00:00:00 2001
From: Elliott Sales de Andrade <quantum.analyst@gmail.com>
Date: Wed, 18 Jan 2023 05:21:23 -0500
Subject: [PATCH] Fix header include in C++ mode

This file is included in the toplevel `xls.h` within an `extern "C"`,
which means including C++ headers in it is completely broken. With GCC
13, this causes strange errors due to "missing" symbols.
--- a/include/libxls/xlstypes.h
+++ b/include/libxls/xlstypes.h
@@ -36,11 +36,7 @@
 #ifndef XLS_TYPES_INC
 #define XLS_TYPES_INC
 
-#ifdef __cplusplus
-#include <cstdint>
-#else
 #include <stdint.h>
-#endif
 #include <sys/types.h>
 
 typedef unsigned char		BYTE;

